noetic/distribution.yaml

```diff
@@ -2483,6 +2483,24 @@ repositories:
       url: https://github.com/ethz-adrl/ifopt.git
       version: master
     status: maintained
+  ifxsensor:
+    doc:
+      type: git
+      url: https://github.com/Yang-IFX/ifxsensor.git
```

Comment:
I noticed one other weird thing when looking through the repository.  https://github.com/Yang-IFX/ifxsensor/tree/master/ifx_bgt60_device/lib/sensor_sdk/libs contains a bunch of pre-built libraries.  The problem with it is that we have no idea whether we have the rights to redistribute those libraries.  Can you talk more about what those libraries are, why they are needed, and where they come from?

Reply:
Hi clalancette, 
first of all, thank you very much for checking all these details. 
For the moment, I can answer your question about:
1. There libraries are build from same source code for different platforms;
2. They are used for config/control the radar chip.
3. The library comes from Infineon. No 3rd party code inside.

So back to your first question: whether you have the rights to redistribute. 
From Infineon side, we are willing to share these binary files to public. That's also the reason we put it on GitHub as public repo for everybody to download. But from legal point of view, if you need some official documents to guarantee the right authorization, please contact me by: Yang.Wu@infineon.com . I am very willing to be supportive here. 
Thanks again.
Best Regards,
Yang

Reply:
One correction:
          **_3. The library comes from Infineon. No 3rd party code inside._**
          Actually there is fftw library we used inside. But Infineon has got the authorization to redistribute it.

Reply:
@clalancette friendly ping, do you think that's ok?

Reply:
Hi @clalancette @ivanpauno , it's a long time since last comment. I just came back from a long vacation. How can we proceed here about this Pull-Request? Thanks very much.

Reply:
@sloretz , could you please take a look at this PR? Thanks very much.

Reply:
> From Infineon side, we are willing to share these binary files to public. That's also the reason we put it on GitHub as public repo for everybody to download. But from legal point of view, if you need some official documents to guarantee the right authorization, please contact me by: Yang.Wu@infineon.com . I am very willing to be supportive here.

It'd be ideal for such legal documents to be distributed along with the binaries. But I'll defer to @clalancette.

Reply:
The problem here is that anything that is not a strict open-source license as defined by OSI (https://opensource.org/licenses) is basically impossible for us.  We don't have any ways to track the provenance of the code, and whether we are allowed to distribute it, and we really don't want to start doing that.  Also, anyone who is consuming our packages has to do the same due diligence, so putting non-open-source code in here makes their life more difficult as well.

My inclination is to close this out, since we just can't be sure of the legality of distributing the code.  But I'll leave one final ping for @tfoote to look at this.

Reply:
I believe that the MIT license can cover software in both source and binary format which is fine for the libraries owned by Infineon. However I know that the license for FFTW is GPL: http://fftw.org/doc/License-and-Copyright.html though I see that they offer commercial licenses. Documentation of that commercial license should be included in a NOTICES file and also noted in the package.xml. Then if all the other content is Infineon and MIT it should be fine.

Reply:
Add a NOTICE file alongside with the binary files to clarify. Thanks.